scripts/west_commands/run_common.py

```diff
@@ -363,17 +372,18 @@ def dump_context(command, args, unknown_args):
     else:
         cls = None
 
-    if runner_config is None:
+    if not runner_configs:
         dump_context_no_config(command, cls)
     else:
         log.inf(f'build configuration:', colorize=True)
         log.inf(f'{INDENT}build directory: {build_dir}')
         log.inf(f'{INDENT}board: {board}')
         log.inf(f'{INDENT}runners.yaml: {runners_yaml}')
-        if cls:
-            dump_runner_context(command, cls, runner_config)
-        else:
-            dump_all_runner_context(command, runner_config, board, build_dir)
+        for runner_config in runner_configs:
+            if cls:
+                dump_runner_context(command, cls, runner_config)
+            else:
+                dump_all_runner_context(command, runner_config, board, build_dir)
```

Comment:
```suggestion
           dump_all_runner_context(command, runner_config, board,
                         build_dir)

```

Borked the indentation, but you get the idea, we need to break the line.

Reply:
fixed.